drivers/soundwire/bus.c

```diff
@@ -1406,12 +1406,14 @@ static int sdw_handle_slave_alerts(struct sdw_slave *slave)
 		 * interrupt
```

Comment:
typo in commit msg: "even the PARITY mask is not set" -> "even if the PARITY mask is not set"

Reply:
nice catch, fixed now.